### PR TITLE
EventStream interfaces and project skeletons

### PR DESCRIPTION
**Pull request by ianandrewson**
# What
Added skeleton for eventStream function and required types. 
Added skeleton test file for eventStream. 
Pseudocode for scoreArray written. 
Ready to write tests then TDD.

### CODE REVIEW

File: src/eventStream.test.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+describe("Event Stream coding challenge tests", () => {
+	it("passes", () => {
+		expect(true).toBeTruthy()
```
Comment (rachoac) [inline src/eventStream.test.ts]: use the formatting tool to avoid excessive space

File: src/eventStream.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
+type EventType = "screenshot" | "new message" | "view"
+
+interface StreamEvent {
+  timestamp: number,
```
Comment (rachoac) [inline src/eventStream.ts]: these should be readonly, to practice immutability

Comment (ianandrewson) [reply src/eventStream.ts]: Ahhh, alright. Will update.

File: src/eventStream.test.ts (hunk 1 above)
Comment (ianandrewson) [reply src/eventStream.test.ts]: Hmmmm I just ran the formatting tool but no changes were made to this file.

Are it statements similar to if statements in that there should be a space after it and before the parenthesis?

**Pull request merged**